pkgs/applications/networking/irc/chatterino2/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, pkgconfig, fetchFromGitHub, qtbase, qtsvg, qtmultimedia, qmake, boost, openssl }:
+
+stdenv.mkDerivation rec {
+  pname = "chatterino2";
+  version = "2.0.4";
+  name = "${pname}-${version}";
+  src = fetchFromGitHub {
+    owner = "fourtf";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1vdln1wfdd1s1prb9iab5spyfv34jvq7p8cwc3313izhbq5c2r05";
+    fetchSubmodules = true;
+  };
+  nativeBuildInputs = [ qmake pkgconfig ];
+  buildInputs = [ qtbase qtsvg qtmultimedia boost openssl ];
+  installPhase = ''
+    start="$(pwd)"
+    mkdir -p $out/bin
+    cp $start/bin/chatterino $out/bin/chatterino
+  '';
+  meta = with stdenv.lib; {
+    description = "A chat client for Twitch chat";
+    longDescription = ''
+  Chatterino is a chat client for Twitch chat. It aims to be an
+  improved/extended version of the Twitch web chat. Chatterino 2 is
+  the second installment of the Twitch chat client series
+  "Chatterino".
+  '';
+    homepage = "https://github.com/fourtf/chatterino2";
+    license = licenses.mit;
+    platforms = [ "x86_64-linux" ];
```

Comment:
There was no mention of  64 bit only support in the README and reviewing the `.pro` makes me think you could use `platforms.unix` here instead.